packages/client-ts/src/utils.ts

```diff
@@ -75,16 +76,24 @@ export function explodeVeridaUri(uri: string): FetchUriParams {
 
 /**
  * Fetch the data accessible from a Verida URI
- * 
+ *
  * @param uri Verida URI of the record to access
  * @param context An existing context used to open the external database
- * @returns 
+ * @returns
  */
-export async function fetchVeridaUri(
-	uri: string,
-	context: Context
-): Promise<string> {
+export async function fetchVeridaUri(uri: string): Promise<string> {
 	const url = explodeVeridaUri(uri);
+	const TESTNET_DEFAULT_DID_SERVER = 'https://dids.testnet.verida.io:5001';
```

Comment:
@cmcWebCode40 You can't do this.

The `client` object needs to be passed in to this method from the application. You can automatically instantiate using the testnet as that won't work in production.